src/controllers/confirmation.controller.ts

```diff
@@ -0,0 +1,22 @@
+import { Session } from "ch-node-session-handler";
+import { UserProfileKeys } from "ch-node-session-handler/lib/session/keys/UserProfileKeys";
+import { NextFunction, Request, Response } from "express";
+import { SESSION_OBJECTION_ID } from "../constants";
+import { Templates } from "../model/template.paths";
+import { retrieveFromObjectionSession, retrieveUserProfileFromSession } from "../services/objection.session.service";
+
+export const get = (req: Request, res: Response, next: NextFunction) => {
+    const session: Session | undefined  = req.session;
+    if (session) {
+        const userProfile = retrieveUserProfileFromSession(session);
+        const objId: string = retrieveFromObjectionSession(session, SESSION_OBJECTION_ID);
```

Comment:
You could call this variable 'objectionId' and then just reference it in the render method below, rather than do 'objectionId: objId'. Just a short-hand way of achieving the same thing. Take a look at the check your answers controller which does something similar.

Reply:
changed this to use variables only